frontend/src/theme/components/Avatar.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/react'
+
+export const Avatar: ComponentMultiStyleConfig = {
+  parts: ['container', 'badge', 'usernameItem', 'usernameIcon'],
+  baseStyle: {
+    container: {
+      backgroundColor: 'primary.500',
+      fontSize: '0.875rem',
+      fontWeight: '500',
```

Comment:
This seems incorrect. I think we should use `textStyle: 'subhead-2'` (according to figma design) instead of redeclaring the text style here so the design changes if we update subhead-2. If the textStyle does not hold, we can import textStyle and spread its props directly in this object.

Reply:
thanks, edited